src/pages/account/your-beacon-registry-account.tsx

```diff
@@ -234,7 +241,10 @@ export const getServerSideProps: GetServerSideProps = withContainer(
     const beacons = await getBeaconsByAccountHolderId(accountHolderDetails.id);
 
     return {
-      props: { accountHolderDetails, beacons },
+      props: {
+        accountHolderDetails,
+        beacons,
+      } as YourBeaconRegistyAccountPageProps,
```

Comment:
What is this type casting for?  I think the returned value matches the interface? (And if it doesn't, is there another problem we need to fix?)